olive/workflows/run/config.py

```diff
@@ -83,33 +116,53 @@ def validate_pass_search(cls, v, values):
             disable_search = v.get("disable_search", False)
 
         v["disable_search"] = disable_search
+        if v["type"] == "OnnxQuantization" or v["type"] == "OnnxStaticQuantization":
```

Comment:
Should we include a flag similar to `Pass._requires_user_script` to auto add the `data_config` `PassConfigParam` into the pass's `default_config`? We can also check for the flag here instead of hard coding it.

Reply:
Agree!  Actually I tried to add that before. Just thought if we have data_config ready then requires_user_script can be deprecated then I removed it somehow. 😿 

Let me add it back. Thanks~!